patches/chromium/render_widget_host_view_base.patch

```diff
@@ -39,19 +39,28 @@ index bdd8712db25a50607999893fe78cd195023704cb..b9c57ea019cbf04c77116f12b9ae1e3a
  #include "content/public/browser/render_widget_host_view.h"
  #include "content/public/common/page_visibility_state.h"
  #include "content/public/common/widget_type.h"
-@@ -70,9 +73,11 @@ namespace content {
+@@ -67,16 +70,18 @@ enum class DomCode : uint32_t;
+ 
+ namespace content {
+ 
++class DelegatedFrameHost;
  class DevicePosturePlatformProvider;
  class MouseWheelPhaseHandler;
  class RenderWidgetHostImpl;
 +class RenderWidgetHostViewGuest;
  class ScopedViewTransitionResources;
++class SyntheticGestureTarget;
  class TextInputManager;
  class TouchSelectionControllerClientManager;
-+class WebContentsView;
+ class TouchSelectionControllerInputObserver;
  class WebContentsAccessibility;
- class DelegatedFrameHost;
- class SyntheticGestureTarget;
-@@ -144,6 +149,10 @@ class CONTENT_EXPORT RenderWidgetHostViewBase
+-class DelegatedFrameHost;
+-class SyntheticGestureTarget;
```

Comment:
Looking at the patch as it stands in main (https://github.com/electron/electron/blob/main/patches/chromium/render_widget_host_view_base.patch), we do not delete/move these class declarations and we shouldn't do that here.

Reply:
Is your concern related to this type of change introducing additional patch conflicts ? I think @ckerr sorted it to avoid such issues but yeah agree that the following two classes should have been sorted in upstream for this to work.

```
class DelegatedFrameHost;
class SyntheticGestureTarget;
```

Reply:
@deepak1556 a couple of things: 1. It has the potential to cause more patch conflicts and 2. Someone looking at the patch may think that there was a necessary reason for the change vs formatting. 3. We should aim to keep our patches as small as possible.